1st-page/index.html

```diff
@@ -0,0 +1,257 @@
+﻿﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Page1 Lesson1 Assignment</title>
+</head>
+
+<style>
+
+    .header {
+        color: white;
+        background-color: #003366;
+        padding: 25px;
+        background-image: url("lesson04-mock2-banner-image.jpg");
+        background-repeat: no-repeat;
+        background-size: 1500px 160px;
+    }
+
+    h3 {
+        color: #4CAF50;
+    }
+
+    .hidden {
+        position: relative;
+        left: -10000px;
+        top: auto;
+    }
+
+        .hidden:focus {
+            left: 0;
+        }
+
+    nav {
+        background-color: #DCDCDC;
+        padding: 30px;
+    }
+
+    .adoptIntro img {
+        float: left;
+        margin-right: 40px;
+        margin-left: 40px;
+    }
+
+    .adoptIntro {
+        margin-bottom: 80px;
+    }
+
+    .flex {
+        display: flex;
+    }
+
+        .flex > section {
+            margin-top: 100px;
+            margin-right: 30px;
+            width: 15%;
+            border: 3px solid #73AD21;
+            padding: 15px;
+        }
+
+    .flexi {
+        display: flex;
+    }
+
+        .flexi > section {
+            margin-top: 100px;
+            margin-right: 30px;
+            width: 15%;
+            border: 3px solid #73AD21;
+            padding: 15px;
+        }
+
+    footer {
+        text-align: center;
+        padding: 80px;
+        background-color: dimgray;
+        color: white;
+        margin-top: 125px;
+        background-image: url("lesson04-mock4-footer-image.jpg");
+    }
+</style>
+
+<body>
+
+    <header>
+
+        <section class="header">
+            <img src="dog-logo.png" alt="dog logo">
+            <h3><i>Seattle's premiere dog adoption agency</i></h3>
+        </section>
+
+        <a href="#main-content" class="hidden">Skip to main contet</a>
+
+        <nav>
+            <a href="index.html">Home</a> /
+            <a href="DogsPage.html">Dogs</a> /
+            <a href="Blog.html">Blog</a> /
+            <a href="Checkout.html">Checkout</a>
+        </nav>
+
+    </header>
+
+    <section class="adoptIntro">
+
+        <img src="home-intro.jpg" alt="Owner hugging golden retrirver"
+             style="height:250px;width:300px;">
```

Comment:
Inline styles are generally bad practice and should be avoided. Also px are not responsive.